src/config.rs

```diff
@@ -45,6 +45,8 @@ pub struct ServerConfig {
     pub timeout: Option<Duration>,
     /// Authentication token.
     pub auth_token: Option<String>,
+    /// Authentication tokens.
+    pub auth_tokens: Option<Vec<String>>,
```

Comment:
How about we deprecate `auth_token` field and only support `auth_tokens`? I think it would help greatly refactoring the code and also having an array is definitely more convenient. What do you think?

Reply:
Hah. Yep, I was thinking about this as well. But then I have noticed something interesting: it seems that this config option was never made public. It's not in the example `config.toml`, nor in the README. IMO it is perfectly fine to deprecate it (and get rid of it when you create a `1.0.0` release in the future), but I still think that the env var should be kept. Especially for the Docker folks who like to configure as much as possible with env vars.

You don't even have to mention the deprecation, because this option was never mentioned in te first place either. Well, this only works, if you keep the env var.

In many cases a single auth token _might_ still be the most prevalent use-case, thus keeping the env var could be a good thing. But it's up to you.

Reply:
I think it was once mentioned in the changelog and release page which is enough for visibility. Also, this was a requested feature in #14 

We should keep the environment variable for sure. I meant `server.auth_token` -> `server.auth_tokens` and showing a log message if `server.auth_token` is used in the configuration file.

Reply:
Yep, this makes perfect sense. I'll add that.

Reply:
Done.